test/bn_spatial_test.cpp

```diff
@@ -1255,7 +1264,7 @@ struct batch_norm_spatial_driver : test_driver
 #if(MIO_BN_SP_TEST_DEBUG == 1)
         std::cout << "Running back propagation spatial recalc." << std::endl;
 #endif
-        this->tolerance = 80 * input.desc.GetElementSize();
+        this->tolerance = 80000 * input.desc.GetElementSize();
```

Comment:
What's reason for adjusting tolerance?  Is it due to smaller batch size?

Reply:
I am second to this question.